build.cmd

```diff
@@ -12,21 +12,41 @@ REM is not supported. Also handle x86 operating systems, where %ProgramFiles(x86
 REM %MSBuild% value when setting the variable and never quote %MSBuild% references.
 set MSBuild="%ProgramFiles(x86)%\MSBuild\12.0\Bin\MSBuild.exe"
 if not exist %MSBuild% @set MSBuild="%ProgramFiles%\MSBuild\12.0\Bin\MSBuild.exe"
+set FullBuild=0
 
-if /I "%1" == "" goto BuildDefaults
+REM with no switches, run the full set of tests.
+if /I "%1" == "" (
```

Comment:
Can we combine this non-switches case with the "FULL" case below since they are equivalent?